logs/logs.go

```diff
@@ -29,6 +29,7 @@ type LogEvent interface {
 // e.g. a single log file
 type LogSrc interface {
 	SetOutput(func(LogEvent))
+	SetFilter(func(LogEvent) bool)
```

Comment:
This is in the interface, so it gets overridden in the tailer log source. Not sure if you wanted it to go somewhere else

Reply:
I mean I don't see the `tailerSrc` implementation being called anywhere either.

Reply:
You're right. The function gets passed in with the construction in `NewTailerSrc`. With the refactoring, I think I'll just rip this out since it's probably not necessary anymore.